### DISCUSSION TITLE
Can't split service id into 2 parts

### DISCUSSION THREAD

**Discussion by sruby** [apache/skywalking#8867] [Q&A] [answered]
OAP version: 9.0.0
**OAP Config**
use default config with H2

**Agent Config**
`-javaagent:skywalking-agent\skywalking-agent.jar=agent.service_name=demo,agent.instance_name=local,agent.namespace=demo,collector.backend_service=x.x.x.x:11800,logging.level=debug
`

when i start my application OAP error， and UI has no information.
**Error Log:**
`org.apache.skywalking.oap.server.core.analysis.worker.MetricsPersistentWorker 273 [pool-6-thread-1] ERROR [] - [9.0.0-89644fe (20220217220813)] Can't split service id into 2 parts, org.apache.skywalking.oap.server.core.UnexpectedException: Can't split service id into 2 parts, at org.apache.skywalking.oap.server.core.analysis.IDManager$ServiceID.analysisId(IDManager.java:56) ~[server-core-9.0.0.jar:9.0.0] at org.apache.skywalking.oap.server.core.alarm.provider.NotifyHandler.notify(NotifyHandler.java:77) ~[server-alarm-plugin-9.0.0.jar:9.0.0] at org.apache.skywalking.oap.server.core.alarm.AlarmEntrance.forward(AlarmEntrance.java:39) ~[server-core-9.0.0.jar:9.0.0] at org.apache.skywalking.oap.server.core.analysis.worker.AlarmNotifyWorker.in(AlarmNotifyWorker.java:41) ~[server-core-9.0.0.jar:9.0.0] at org.apache.skywalking.oap.server.core.analysis.worker.AlarmNotifyWorker.in(AlarmNotifyWorker.java:30) ~[server-core-9.0.0.jar:9.0.0] at org.apa
`

**Comment by wu-sheng** [comment, marked as answer]
Read https://github.com/apache/skywalking/issues/8864
